build.sh

```diff
@@ -22,11 +22,19 @@ test -d node_modules && rm -rf node_modules
 npm install
 ./do compile:all --env production
 
+# Dependency injection container dump.
+echo '[BUILD] Dumping dependency injection container'
```

Comment:
Again: `Dumping dependency injection container` doesn't say anything. Should it be something like: `Creating dependency injection container cache`?